packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactStylesDiffMap.kt

```diff
@@ -60,4 +60,10 @@ public class ReactStylesDiffMap(props: ReadableMap) {
   public fun getDynamic(name: String): Dynamic = backingMap.getDynamic(name)
 
   override fun toString(): String = "{ ${javaClass.simpleName}: $backingMap }"
+
+  /**
+   * Access the package private property for faster access to the underlying values.
+   This is used in Expo to override setting properties in some subclassed view-managers
+  */
+  public fun getBackingMap(): ReadableMap = backingMap
```

Comment:
I won't do that. Instead I would suggest we either:

1. Have a 
```kotlin
/** This backing field is access by Expo to do bla bla bla */
@JvmField 
@FrameworkApi
public val internal_backingMap: ReadableMap = props
```

2. Even better:

```kotlin
/** This backing field is access by Expo to do bla bla bla */
@JvmField
@JvmName("internal_backingMap") 
@FrameworkApi
internal val backingMap: ReadableMap = props
```

You will then be able to access this backing field as `ReactStylesDiffMap.internal_backingMap` ONLY from a Java class inside Expo.

I also think in general we should refactor this (I remember taking about it with @lukmccall back in the day)

Reply:
We're accessing this through kotlin classes, so the above might not work?

Reply:
I did a quick stunt to see if it could be refactored in the Expo code, and we can do that - but it causes a potential performance regression since we'd need to convert back and forth between a lot.